src/editors/playgroundController.ts

```diff
@@ -666,19 +686,20 @@ export default class PlaygroundController {
 
     const selections = this._activeTextEditor.selections;
 
+    let codeToEvaluate;
     if (
       !selections ||
       !Array.isArray(selections) ||
       (selections.length === 1 && this._getSelectedText(selections[0]) === '')
     ) {
```

Comment:
I know this is code that didn't change in this PR, but it's counterintuitive to me that we need to do these evaluations here, considering we already have `this._selectedText`. I would expect that this condition is equivalent to `!this._selectedText`.